### ISSUE TITLE
error in command

### ISSUE THREAD

**Issue by averied** [MicrosoftDocs/dynamics365smb-devitpro-pb#2790]
I guess this is a mistake:

Sync-NavApp -ServerInstance $BcServerInstance -Tenant $TenantId -Name Application -Tenant $TenantId

In stead of twice having the -Tenant, you are missing the -Version parameter

---
#### Detalles del documento

⚠ *No edite esta sección. Se requiere para learn.microsoft.com ➟ Vinculación de problema de GitHub.*

* ID: 1c6e3f3d-2927-52e9-f629-59c4f17c5b38
* Version Independent ID: 1c6e3f3d-2927-52e9-f629-59c4f17c5b38
* Content: [Install a version 20 update - Business Central](https://learn.microsoft.com/es-es/dynamics365/business-central/dev-itpro/upgrade/upgrading-cumulative-update-v20)
* Content Source: [dev-itpro/upgrade/upgrading-cumulative-update-v20.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/upgrade/upgrading-cumulative-update-v20.md)
* Service: **dynamics365-business-central**
* GitHub Login: @jswymer
* Microsoft Alias: **jswymer**

**Comment by jswymer** [comment]
Good catch! It's been fixed to include the -Version. The update will be published soon. Thanks for the feedback.
